src/options/options.html

```diff
@@ -87,6 +87,7 @@ <h1 data-i18n="">Unicode autocorrection</h1>
 				</ul>
 			</section>
 
+			<!-- Remove "mobile-incompatible" once https://bugzilla.mozilla.org/show_bug.cgi?id=1595822 is fixed -->
```

Comment:
Same here…
Do you “abuse” this mobile-incompatible blacklisting for Thunderbird? :upside_down_face: 

If so, that’s of course not the proper way to do so. Because this add-on may (as for the auto-correction part, at least), run fine on Firefox for Android (Fenix, if they allow all add-ons in a future version). So we need a different `thunderbird-incompatible` CSS and `isThunderbird` JS or so.

Reply:
See above. The context menu does not currently work in the compose window of Thunderbird because of [bug 1656506](https://bugzilla.mozilla.org/show_bug.cgi?id=1656506), although it does not need to be disabled and it looks like this may be fixed by [bug 1670832](https://bugzilla.mozilla.org/show_bug.cgi?id=1670832).